modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,29 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== In Safari running on iOS, Hangul characters merged into the previous line upon typing after pressing Enter.
+//#TINY-9746
+
+Previously, in Safari running on iOS, when a newline was inserted (by pressing the Enter key) and the user continued typing without moving the insertion point, the characters typed on the newline merged onto the previous line and the newline was removed.
+
+This occurred because Safari on iOS composes Hangul syllabic blocks by deleting the individual jaso glyphs and inserting the expected block glyph, continuing until a native Enter `keypress` event is fired.
```

Comment:
```suggestion
This occurred because Safari on iOS composes Hangul syllabic blocks by deleting the individual jaso glyphs and inserting the expected block glyph, continuing until a browser-native Enter `keypress` event is fired.
```